apps/server/src/modules/learnroom/service/course.service.ts

```diff
@@ -18,11 +21,13 @@ export class CourseService {
 	}
 
 	public async deleteUserDataFromCourse(userId: EntityId): Promise<number> {
+		this.logger.log({ action: 'Deleting data from Courses for user ', userId });
 		const [courses, count] = await this.repo.findAllByUserId(userId);
 
 		courses.forEach((course: Course) => course.removeUser(userId));
 
 		await this.repo.save(courses);
+		this.logger.log({ action: 'Deleting data from Courses for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of updated courses without userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114